src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -65,17 +65,18 @@ public function generate( $params ) {
 	 * @param $params
 	 */
 	protected function set_params( $params ) {
-		$this->params = shortcode_atts( array(
-			'limit' => -1,
-			'user'  => get_current_user_id()
-		), $params, $this->shortcode_name );
+		$this->params = shortcode_atts( [
+			'limit' => - 1,
+			'user'  => get_current_user_id(),
+		], $params, $this->shortcode_name );
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: ddda6b8d4e6a3eb14624620ff8ffa89e</sub>